webpack.config.ts

```diff
@@ -47,13 +47,13 @@ export default (env: { standalone?: boolean } = {}) => ({
     fallback: {
       path: require.resolve('path-browserify'),
       http: false,
-      fs: false,
-      os: false,
+      fs: env.browser ? path.resolve(__dirname, 'src/empty.js') : false,
```

Comment:
I believe we don't need this conditional here, just keep it empty. It will be overwritten by `externalsPresets` anyways.